exercises/isogram/example.h

```diff
@@ -0,0 +1,7 @@
+#include <string>
+
+namespace isogram {
+
+bool is_isogram(std::string const);
```

Comment:
Should use a constant string reference here.

(In C++17 we'd use a string_view, but we need to support earlier versions)

Reply:
OK, done.